buildings/gui/alter_building_relationships.py

```diff
@@ -37,6 +41,7 @@ def __init__(self, dockwidget, current_dataset, parent=None):
 
         self.dockwidget = dockwidget
         self.layer_registry = LayerRegistry()
+        self.lyr_bulk_load = QgsVectorLayer()
```

Comment:
This is not necessary as long as `self.edit_dialog = EditDialog(self)` is after `self.layer_setup()`